### DISCUSSION TITLE
How to make a table with two row selection in react-table

### DISCUSSION THREAD

**Discussion by ghost** [TanStack/table#4497] [Q&A]
I need to make a table with two multi-select columns that will be separate from each other, and now when I add a second column they are still linked together, you can see this in the example by clicking on the checkbox https://codesandbox.io/s/xenodochial-architecture-5c2rqr?file=/src/App.js

**Comment by FeShimada** [comment]
Same here, I need to make a table with 2 column (first checkbox and other toggle button) but apparently tanstack does not support this 2 states management system
